Sources/App/Models/Build.swift

```diff
@@ -0,0 +1,63 @@
+import Fluent
+import Vapor
+
+
+final class Build: Model, Content {
+    static let schema = "builds"
+
+    typealias Id = UUID
+
+    // managed fields
+
+    @ID(key: .id)
+    var id: Id?
+
+    @Timestamp(key: "created_at", on: .create)
+    var createdAt: Date?
+
+    @Timestamp(key: "updated_at", on: .update)
+    var updatedAt: Date?
+
+    // reference fields
+
+    @Parent(key: "version_id")
+    var version: Version
+
+    // data fields
+
+    @Field(key: "logs")
+    var logs: String?
+
+    @Field(key: "platform")
+    var platform: Platform?
```

Comment:
This probably shouldn't be the current `Platform` enum, but a new one for builds.

Reply:
Fixed! As to the actual values of the enum - that's probably going to change a bit but it's a simple, non-schema affecting change.